server/src/items/classes/spell_scrolls/PacifySpellScroll.js

```diff
@@ -0,0 +1,34 @@
+const SpellScroll = require("./SpellScroll");
+const ProjPacify = require("../../../entities/classes/destroyables/movables/projectiles/ProjPacify");
+
+class PacifySpellScroll extends SpellScroll {
+    onUsed() {
+        
+        console.log("Pacify!");
+
+        const
+            { row } = this.owner;
+        const { col } = this.owner;
```

Comment:
These destructuring lines can be combined into `const {row, col, board} = this.owner`;